### PR TITLE
development workflow

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: _posts/development-guide/2021-05-17-dev-workflow.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+When developing a new scene, you use the `@latest` stable SDK release by default.
+
+You can install the `@next` SDK release if you want to leverage or preview upcoming features that didn't yet make it into the latest stable release.
+
+To do so, run the following on your scene:
+
+`npm i decentraland-ecs@next`
+
+> Note: Keep in mind that the @next version might suffer issues from time to time. The syntax of new features might change before it's released in a stable version.
```
Comment (menduz) [inline _posts/development-guide/2021-05-17-dev-workflow.md]:
```suggestion
> Note: Keep in mind that the @next version might suffer issues from time to time. The syntax and name of new features might change before it's released in a stable version.
```

File: _posts/development-guide/2021-05-17-dev-workflow.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+```bash
+dcl start
+```
+
+See [Preview your scene]({{ site.baseurl }}{% post_url /development-guide/2018-01-04-preview-scene %}) ) for more details. Check the [Debug a scene]({{ site.baseurl }}{% post_url /development-guide/2018-01-04-preview-scene %}#debug-a-scene)) for tips on how to debug any issues.
+
+## Deploy to the test environment
+
+Deploy your scene to the test server.
```
Comment (nchamo) [inline _posts/development-guide/2021-05-17-dev-workflow.md:37]: Would it make sense to explain that anyone can set up and use their own catalyst if they want to?

**Review by nchamo — APPROVED** [review]
Great addition!

File: _posts/development-guide/2021-05-17-dev-workflow.md (hunk 2 above)
Comment (nearnshaw) [reply _posts/development-guide/2021-05-17-dev-workflow.md:37]: cool yeah! I like the idea, would be worth it for big projects that want to keep secret till launch

is there any readme for that I can link to @nchamo ?

Comment (nchamo) [reply _posts/development-guide/2021-05-17-dev-workflow.md:37]: The repo's README has detailed instructions on how to do it

https://github.com/decentraland/catalyst-owner/

**Pull request merged**